Data/Uniques/body.lua

```diff
@@ -74,6 +74,16 @@ Implicits: 1
 {variant:2,3}You take 450 Chaos Damage per second for 3 seconds on Kill
 Gore Footprints
 ]],[[
+Doppelgänger Guise
+Sadist Garb
+League: Ritual
+Requires Level 68, 103 Dex, 109 Int
+Grants Level 20 Unhinge Skill
+(40–60)% more Critical Strike Chance while Insane
```

Comment:
I haven't checked all items, but at least on this line there's an em dash instead of a minus. Please be careful with the formatting when you copy items from poedb.